Objects/typeobject.c

```diff
@@ -7278,6 +7300,9 @@ update_all_slots(PyTypeObject* type)
 {
     slotdef *p;
 
+    /* Clear the VALID_VERSION flag of 'type' and all its subclasses. */
+    PyType_Modified(type);
```

Comment:
Yes, Serhiy and me agreed that it would be a good idea to move this out of `update_slot()` because it needs to be called unconditionally, and currently it's also called repeatedly for each slot in `update_all_slots()`. Now, both callers of `update_slot()` call it directly, unconditionally and only once.

Reply:
Thanks.